src/lib/cloudinary.js

```diff
@@ -235,6 +237,10 @@ async function updateHtmlImagesToCloudinary(html, options = {}) {
     }
 
     $img.setAttribute('src', cloudinaryUrl);
+    if (loadingStrategy) {
+      $img.setAttribute('loading', 'lazy');
```

Comment:
the value being set for the attribute should be the `loadingStrategy` variable itself, it should pass through